packages/components/src/Dialog/Dialog.component.js

```diff
@@ -50,16 +71,16 @@ Dialog.defaultProps = {
 
 Dialog.propTypes = {
 	header: PropTypes.string,
-	size: PropTypes.oneOf(['small', 'large']),
+	size: PropTypes.oneOf(['sm', 'small', 'lg', 'large']),
```

Comment:
why have 'sm' and 'small' (same for 'lg' and 'large') ?

Reply:
one alone is not enough?

Reply:
yes they are the same